src/components/microlearning/VideoDetailPage.jsx

```diff
@@ -61,7 +74,7 @@ const VideoDetailPage = () => {
       />
     );
   }
-
+  const levelType = getLevelType(intl, courseMetadata.activeCourseRun.levelType);
```

Comment:
NIT
`const levelType = courseMetadata.activeCourseRun.levelType ? getLevelType(intl, courseMetadata.activeCourseRun.levelType) : null;`

Add a check for `levelType` in the code below instead of `courseMetadata.activeCourseRun.levelType`